JBQA-12612 - Test JDG70.HadoopIOFormat

andyuk1986 opened the pull request:
Added Jacoco plugin to the new profile for calculating the code coverage;
Added more tests for increasing code coverage;
Added tests verifying the failover of dif. servers in the cluster.

gustavonalle commented:
Hi @andyuk1986 , thanks for providing those tests!
I wonder if it's possible to trim them down a bit; this PR slows down the suite 3x to 4x times compared to master, and apparently is using more than 4GB of memory causing CI to struggle.
Would it be possible to use 2 servers instead of 4 and less data than 200k entries, maybe 100?

Also, is ```FailoverHandlingTest``` suppose to fail, since I noticed you opened ISPN-6087 & ISPN-6086 and referenced it? They always pass for me when running locally

andyuk1986 commented:
Hi @gustavonalle, thank you for your review. 

I am sorry I have performed hard reset on my branch as mixed up some commits and this pull request was automatically closed. I have sent new pull request for the same branch. 

I have changed the number of the servers to 2 and the number of entries to 100K, also removed the data writing to HDFS as it was unnecessary. Now I think it should be ok on CI.

For the FailoverHandlingTest - originally it should pass, but the tests are failing as described in ISPN-6087 and ISPN-6085. I ran the tests on my old T520 and there they were failing with stopping the server in 4 seconds. I have decreased the waiting time to 1200msc and now the tests are failing on my new W541 as well.
